### PR TITLE
Terraform Plan and Apply commands

### PR DESCRIPTION
**Pull request by jasonBirchall**
This PR adds terraform plan and apply commands with sensitive data redaction

### CODE REVIEW

File: doc/cloud-platform_terraform_apply.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+### Options
+
+```
+      --aws-access-key-id string       Access key id of service account to be used by terraform
+      --aws-region string              [required] aws region to use
+      --aws-secret-access-key string   Secret access key of service account to be used by terraform
+  -h, --help                           help for apply
+      --redact                         Redact the terraform output before printing (default true)
+      --terraform-version string       [optional] the terraform version to use. (default "1.2.5")
+      --workdir string                 [optional] the terraform working directory to perform terraform operation [default] . (default ".")
```
Comment (poornima-krishnasamy) [inline doc/cloud-platform_terraform_apply.md:18]: This input --workdir seems not used anymore. Can u remove it

File: doc/cloud-platform_terraform_apply.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  -w, --workspace string               Default workspace where terraform is going to be executed
```
Comment (poornima-krishnasamy) [inline doc/cloud-platform_terraform_apply.md:19]: Can you add [required] to the description so we know this is a mandatory field.
Alternatively, u can mark this as Required `cmd.MarkPersistentFlagRequire("workdir")`

File: doc/cloud-platform_terraform_apply.md (hunk 1 above)
Comment (sj-williams) [reply doc/cloud-platform_terraform_apply.md:18]: Can remove, but do you think there might be some use for it testing locally?

**Comment by sj-williams** [comment]
This PR adds terraform plan and apply commands with sensitive data redaction

File: doc/cloud-platform_terraform_apply.md (hunk 1 above)
Comment (poornima-krishnasamy) [reply doc/cloud-platform_terraform_apply.md:18]: If that input is not referred anywhere, then it wont be used for testing

Comment (sj-williams) [reply doc/cloud-platform_terraform_apply.md:18]: Removed

**Review by poornima-krishnasamy — APPROVED** [review]

**Pull request merged**